src/component/noteCard/NoteCard.jsx

```diff
@@ -19,24 +25,15 @@ export const NoteCard = ({ item, archive }) => {
 
   //delete
   const handleDelete = async () => {
-    const deletResponse = await axios.delete(`/api/notes/${id}`, {
-      headers: { authorization: token },
-    });
+    const deletResponse = await deleteNote(id, token);
     if (deletResponse.status === 200 || deletResponse === 201) {
       setNotesList([...deletResponse.data.notes]);
     }
```

Comment:
Suggestion: You could consider adding an else case for error handling. Either add an error screen or an error toast

Reply:
You haven't logged in that's why the note functionality isn't working😅

Reply:
Ok will work on it